Modules/Core/Transform/include/itkComposeScaleSkewVersor3DTransform.hxx

```diff
@@ -306,71 +306,113 @@ ComposeScaleSkewVersor3DTransform<TParametersValueType>::PrintSelf(std::ostream
 template <typename TParametersValueType>
 void
 ComposeScaleSkewVersor3DTransform<TParametersValueType>::ComputeJacobianWithRespectToParameters(
-  const InputPointType & itkNotUsed(p),
-  JacobianType &         itkNotUsed(jacobian)) const
+  const InputPointType & p,
+  JacobianType &         jacobian) const
 {
-  // This document how the jacobian can be computed for certain parameters;
-  //   however, the proper computation of the scale and skew parameters is
-  //   now given or computed, so ultimately an exception is thrown.
+  // Jacobian computed via
+  // pip install pysym
+  // from sympy import Function, Derivative, simplify, Transpose
+  // from sympy import symbols from sympy.matrices import Matrix, eye, zeros, ones, diag
+  // x0, x1, x2, v0, v1, v2, w, o0, o1, o2, s0, s1, s2, k0, k1, k2 = symbols('x0 x1 x2 v0 v1 v2 w o0 o1 o2 s0 s1 s2 k0
+  // k1 k2') M = Matrix([[1-2*(v1*v1+v2*v2), 2*(v0*v1-v2*w), 2*(v0*v2+v1*w), o0],
```

Comment:
line break here?

Reply:
Why is matrix `M` defined this way?

Reply:
It should be the quaternion to rotation matrix conversion: https://www.euclideanspace.com/maths/geometry/rotations/conversions/quaternionToMatrix/index.htm

Reply:
Adding that link here as a comment would be quite helpful.

Reply:
I think stating "quaternion as matrix" someplace would  be sufficcient documentation. I'm not sure if this is the correct place for the python code though.

Reply:
If not here, this code could go into the commit message.

Reply:
The nice part is that this code generates the jacobian in symbolic form, so I'd like to preserve it.  We can also use an adaption of this code to compute jacobians for any of our other transforms that need it.    For that reason, perhaps it is better if the code stays with the jacobian it generated, rather than moving it to a commit message?

Reply:
Will add the link as requested.